staging/2024/euc_counties.csv

```diff
@@ -1,820 +1,337 @@
-state_code,county_name,pa,dr
```

Comment:
Looked at the processing code and it only expects state_code and county_name so that's why pa and dr are removed here.